lib/galaxy/config/__init__.py

```diff
@@ -276,14 +276,17 @@ def convert_datatype(key, value):
         def strip_deprecated_dir(key, value):
             resolves_to = self.schema.paths_to_resolve.get(key)
             if resolves_to:  # value is a path that will be resolved
-                first_dir = value.split(os.sep)[0]  # get first directory component
-                if first_dir == self.deprecated_dirs.get(resolves_to):  # first_dir is deprecated for this option
-                    ignore = first_dir + os.sep
-                    log.warning(
-                        "Paths for the '%s' option are now relative to '%s', remove the leading '%s' "
-                        "to suppress this warning: %s", key, resolves_to, ignore, value
-                    )
-                    return value[len(ignore):]
+                paths = value.split(',')
+                for i, path in enumerate(paths):
```

Comment:
```suggestion
                for i, path in enumerate(paths):
                    path = path.strip()
```

Reply:
We need the strip here, otherwise it'll only work with CSV values that have no white space.
(or just add to a list comprehension in `paths = value.split(',')`

Also, would you mind adding a unit test? I found this by adding this to `test/unit/config/test_path_resolves_to.py`:
```python
def test_kwargs_relative_path_old_prefix_csv_value(mock_init):
    config = BaseAppConfiguration(path4='old-config/foo/file1, old-config/foo/file2,  foo/file3')
    assert config.path4 == ['my-config/foo/file1','my-config/foo/file2', 'my-config/foo/file3']
```

Reply:
Both done, thanks @ic4f !